uber/templates/groupform.html

```diff
@@ -87,6 +87,7 @@
                 </select>
             </div>
             <div class="col-sm-offset-3 col-sm-9" id="table_prices">{{ table_prices() }}</div>
+            <p class="col-sm-9 col-sm-offset-3 help-block" id="moretables"><i>You may contact us via <a href='{{ c.CONTACT_URL }}'>{{ c.CONTACT_URL }}</a> to request more than {{ c.MAX_TABLES }} tables.</i></p>
```

Comment:
This block was originally removed for MAGWest, so we now hide it for MAGWest (https://github.com/magfest/magwest/pull/80).